### DISCUSSION TITLE
it doesnt work with digitalocean

### DISCUSSION THREAD

**Discussion by Nawafwabs** [angristan/wireguard-install#227] [Help]
after setup everything gives me 0 B  received no internet

**Comment by cergo666** [comment]
Same on Oracle cloud

**Comment by ghost** [comment]
Did you check your firewall settings? If you are on Ubuntu, "ufw status" to see whether your VPN's port is on the list? If not, "ufw allow <PORT_NUM>" to enable it.
Other than that, some VPS providers may require to manually enable firewall from their portal.
